en/docs/integrate/develop/generate-docker-image.md

```diff
@@ -21,7 +21,7 @@ See the topics given below.
     
     -  **Create a new Docker Exporter Project**
 
-        Select this option to create a new **Docker Exporter Project** and click **Proceed**. You can build a docker image using this Docker Exporter Project. You are now directed to the [Docker Exporter Project wizard](create-docker-project.
+        Select this option to create a new **Docker Exporter Project** and click **Proceed**. You can build a docker image using this Docker Exporter Project. You are now directed to the [Docker Exporter Project wizard](create-docker-project.md).
```

Comment:
As far as I recall due to the customizations in APIM, if anyone were to define a link ending with `.md`, it will will not work. 
@shaniR , can you please confirm my comment.